Sources/TuistGenerator/Generator/LinkGenerator.swift

```diff
@@ -273,7 +266,13 @@ final class LinkGenerator: LinkGenerating {
             precompiledEmbedPhase.inputPaths = script.inputPaths
                 .map { "$(SRCROOT)/\($0.pathString)" }
             precompiledEmbedPhase.outputPaths = script.outputPaths
+
+            pbxproj.add(object: precompiledEmbedPhase)
+            pbxTarget.buildPhases.append(precompiledEmbedPhase)
         }
+
+        pbxproj.add(object: embedPhase)
+        pbxTarget.buildPhases.append(embedPhase)
```

Comment:
Shouldn't this be done only inside the if? If yes, I would make that `if` a `guard` instead

Reply:
Actually there are 2 build phases in here. `embedPhase` and `precompiledEmbedPhase`. The one needs checking is `precompiledEmbedPhase` and it is added in if statement.